src/Microsoft.FeatureManagement/FeatureManager.cs

```diff
@@ -44,51 +43,208 @@ private class ConfigurationCacheItem
             public object Settings { get; set; }
         }
 
+        /// <summary>
+        /// Creates a feature manager.
+        /// </summary>
+        /// <param name="featureDefinitionProvider">The provider of feature flag definitions.</param>
+        /// <param name="options">Options controlling the behavior of the feature manager.</param>
+        /// <exception cref="ArgumentNullException">Thrown if <paramref name="featureDefinitionProvider"/> is null.</exception>
+        /// <exception cref="ArgumentNullException">Thrown if <paramref name="options"/> is null.</exception>
         public FeatureManager(
             IFeatureDefinitionProvider featureDefinitionProvider,
-            IEnumerable<IFeatureFilterMetadata> featureFilters,
-            IEnumerable<ISessionManager> sessionManagers,
-            ILoggerFactory loggerFactory,
-            IOptions<FeatureManagementOptions> options,
-            IOptions<TargetingEvaluationOptions> assignerOptions)
+            FeatureManagementOptions options)
         {
-            _featureDefinitionProvider = featureDefinitionProvider;
-            _featureFilters = featureFilters ?? throw new ArgumentNullException(nameof(featureFilters));
-            _sessionManagers = sessionManagers ?? throw new ArgumentNullException(nameof(sessionManagers));
-            _logger = loggerFactory.CreateLogger<FeatureManager>();
-            _assignerOptions = assignerOptions?.Value ?? throw new ArgumentNullException(nameof(assignerOptions));
+            _featureDefinitionProvider = featureDefinitionProvider ?? throw new ArgumentNullException(nameof(featureDefinitionProvider));
+            _options = options ?? throw new ArgumentNullException(nameof(options));
             _filterMetadataCache = new ConcurrentDictionary<string, IFeatureFilterMetadata>();
             _contextualFeatureFilterCache = new ConcurrentDictionary<string, ContextualFeatureFilterEvaluator>();
-            _options = options?.Value ?? throw new ArgumentNullException(nameof(options));
-            _parametersCache = new MemoryCache(new MemoryCacheOptions());
+            _featureFilters = Enumerable.Empty<IFeatureFilterMetadata>();
+            _sessionManagers = Enumerable.Empty<ISessionManager>();
+            _telemetryPublishers = Enumerable.Empty<ITelemetryPublisher>();
+            _assignerOptions = new TargetingEvaluationOptions();
         }
 
-        public IEnumerable<ITelemetryPublisher> TelemetryPublishers { get; init; }
+        /// <summary>
+        /// The collection of feature filter metadata.
+        /// <exception cref="ArgumentNullException">Thrown if it is set to null.</exception>
```

Comment:
Isn't exception supposed to be outside of the summary element? Can you check the generated docs?